go/vt/sqlparser/comments.go

```diff
@@ -186,6 +192,13 @@ func StripComments(sql string) string {
 	return sql
 }
 
+func containsQuotations(sql string, start, end int) bool {
+	if start-1 < 0 || end+2 >= len(sql) {
+		return false
+	}
+	return (sql[start-1] == '\'' && sql[end+2] == '\'') || (sql[start-1] == '"' && sql[end+2] == '"')
```

Comment:
as start-1 and end+2 are used at 3 places. Can you place them under vairable and give them a meaningful name which can help understand this return condition?